### PR TITLE
modalai_esc: maintenance release 1

### PR DESCRIPTION
**Pull request by modaltb**
- updated turtle mode to better reflect user experience expectations
- abstract default UART ESC port to boards file
- fix bug with feedback parsing

### CODE REVIEW

File: boards/modalai/fc-v1/src/board_config.h [hunk 1]
```diff
@@ -263,6 +263,8 @@
 
 #define BOARD_NUM_IO_TIMERS 5
 
+// J4 / TELEM3 / UART2
+#define MODALAI_ESC_DEFAULT_PORT                  "/dev/ttyS1"
```
Comment (dagar) [inline boards/modalai/fc-v1/src/board_config.h:267]: We should probably replace this with configurable serial parameters (but it's fine like this for now).

**Review by dagar — APPROVED** [review]

**Comment by dagar** [comment]
@modaltb github won't let me merge this one (claims there are conflicts). Could you try squashing to one commit that's updated on current main?

**Comment by modaltb** [comment]
Hey sorry I spent a while trying to squash on the original branch and it was not happy on my end....  opened a new PR here:
https://github.com/PX4/PX4-Autopilot/pull/20225

File: boards/modalai/fc-v1/src/board_config.h (hunk 1 above)
Comment (modaltb) [reply boards/modalai/fc-v1/src/board_config.h:267]: agreed, will add to my todos

**Pull request closed without merging**